x-pack/plugins/fleet/public/components/agent_enrollment_flyout/steps.tsx

```diff
@@ -30,9 +39,16 @@ export const DownloadStep = () => {
             defaultMessage="Fleet Server runs on an Elastic Agent. You can download the Elastic Agent binaries and verification signatures from Elastic’s download page."
           />
         </EuiText>
+        <EuiSpacer size="s" />
+        <EuiText size="s">
+          <FormattedMessage
+            id="xpack.fleet.agentEnrollment.downloadUseLinuxInstaller"
+            defaultMessage="Linux users: We recommend using the installers over (RPM/DEB) because they provide the ability to upgrade your agent within Fleet."
+          />
+        </EuiText>
         <EuiSpacer size="l" />
         <EuiButton
-          href="https://ela.st/download-elastic-agent"
+          href={`https://www.elastic.co/downloads/past-releases/elastic-agent-${kibanaVersionURLString}`}
```

Comment:
one thing that will be annoying, is that this link will be broken for releases which are not published yet (i.e. local development from master/8.0.0, build candidates like the upcoming 7.14.0)

but I'm not sure what we can do about that. @mostlyjason WDYT?

Reply:
I was concerned about that too, but there doesn't seem to be an easy way to determine in the code whether or not a version is released yet. This might just have to be a thing we deal with in development.

Reply:
sgtm 👍🏻